onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -5815,10 +5815,44 @@ def aten_type_as(self: TensorType, other: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_unfold(self: TensorType, dimension: int, size: int, step: int) -> TensorType:
+@torch_op("aten::unfold", trace_only=True)  # FIXME: Seems ast.For was not supported
+def aten_unfold(self: TTensor, dimension: int, size: int, step: int) -> TTensor:
```

Comment:
I don't understand what the op is supposed to do. Is there a description I can read somewhere? Looks like it is a variant of a slice-like op with params (size, step) in (dimension), followed by some form of transpose?

Reply:
The unfold do below things:
x = [1,2,3,4,5,6], 
unfold it to [1,2],[2,3],[3,4]... when stride=2,step=1
unfold it to [1,2],[3,4],[5,6] when stride=2,step=2
it is a Core aten op.